src/helpers/byTestId.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import { makeQueries } from './makeQueries';
+import type { Queries } from './makeQueries';
+
+const getNodeByTestId = (node, testID) => {
```

Comment:
Consider adding explicit types

Reply:
We can address that in a followup if necessary (I suspect Flow infers that, as it's internal fn)

Reply:
I confirm, flow infers it 
![image](https://user-images.githubusercontent.com/32499425/104781532-e8523a00-5782-11eb-9972-fa86c293896d.png)